client/src/pages/About/About.css

```diff
@@ -0,0 +1,46 @@
+html, body {
+    height: 100%;
+    font-family: Cabin;
+}
+
+.container-fluid {
+    height: 600px;
+    background-image: url(../../assets/uclaheader.png);
+    /* This is what makes the background image rescale based on the container's size */
+    background-size: cover;
+
+    /* Background image is centered vertically and horizontally at all times */
+    background-position: center center;
+  
+    /* Background image doesn't tile */
+    background-repeat: no-repeat;
+    
+    /* Background image is fixed in the viewport so that it doesn't move when 
+        the content's height is greater than the image's height */
+    background-attachment: fixed;
+    
+    /* Set a background color that will be displayed
+        while the background image is loading */
+    background-color: #464646;
+    background-color: rgba(244, 207, 76, 0.4);
+    background-blend-mode: multiply;
+}
+
+.text {
+    font-family: Cabin;
```

Comment:
Make sure to import the font in App.css, or else it just uses a default font. 

eg, change line 2 of app.css to: `@import url('https://fonts.googleapis.com/css?family=Roboto|Raleway|Cabin');`.